docs: update ctrl-c section blogpost (#5255)
aws/copilot-cli: site/content/blogs/release-v130.en.md

Copilot v1.30 brings big enhancements to help you develop more flexibly and efficiently:

- **`copilot run local`**: Copilot has a new operational command that enables you to run services locally! [See detailed section](#copilot-run-local).
- **Roll back deployments with Ctrl-C**: The wait is over! Roll back your CloudFormation deployment right from the terminal, whenever you want. [See detailed section](#roll-back-deployments-with-ctrl-c).
- **Pre- and post- deployment pipeline actions**: Insert db migrations, integration tests, and/or other actions before or after workload/environment deployments. [See detailed section](#deployment-actions). 
- **`copilot deploy` enhancements**: We've increased the scope and flexibility of this command. [See detailed section](#copilot-deploy-enhancements).

> - **`detach flag`**: Skip progress of CloudFormation stack events on your terminal. [see detailed section](#use---detach-to-deploy-without-waiting).

Reviewer: ```suggestion
- **`--detach flag`**: Skip progress of CloudFormation stack events on your terminal. [see detailed section](#use---detach-to-deploy-without-waiting).
```

## `copilot run local`

## Roll back deployments with Ctrl+C

> While waiting for your Copilot service, environment, or job to deploy, You can now hit `Ctrl-C` to rollback your current CloudFormation stack to previous configuration.
> If you hit 'Ctrl-C' a second time, the program will exit, but the stack rollback will continue.
> 
> If you are deploying a resource for the first time, hitting `Ctrl-C` during deployment will delete the stack.
> If you hit 'Ctrl-C' a second time, the program will exit, the deletion of stack will continue.

Reviewer 2: nit nit:
```suggestion
While waiting for your service, job, or environment to deploy, you can now hit `Ctrl-C` to cancel the update. This will either rollback your stack to its previous configuration, or delete the stack if it was being deployed for the first time.

If you hit 'Ctrl-C' a second time, the program will exit, but the stack rollback/deletion will continue.

```

## `copilot run local`

## Roll back deployments with Ctrl-C

While waiting for your service, job, or environment to deploy, you can now hit `Ctrl-C` to cancel the update. This will either rollback your stack to its previous configuration, or delete the stack if it was being deployed for the first time.

If you hit 'Ctrl-C' a second time, the program will exit, but the stack rollback/deletion will continue.

> 

Reviewer: ```suggestion
```

> ## Ctrl-C

Reviewer: could we maybe change this to "Roll back deployments with Ctrl-C"? (or something, just slightly more descriptive)

Author: I changed it to `Roll back deployments with Ctrl-C`

> When your Copilot Service is failing to deploy, You can now hit `Ctrl-C` to rollback your current CloudFormation stack to previous configuration.

Reviewer: ```suggestion
While waiting for your Copilot service, environment, or job to deploy, You can now hit `Ctrl-C` to rollback your current CloudFormation stack to previous configuration.
```

> If you hit 'Ctrl-C' again during a rollback, the rollback of CloudFormation stack will continue in the background.

Reviewer: ```suggestion
If you hit 'Ctrl-C' a second time, the rollback of CloudFormation stack will continue in the background.
```

Reviewer 2: piggybacking on danny's suggestion - 
```
If you hit 'Ctrl-C' a second time, the program will exit, but the stack rollback will continue.
```

"in the background" sort of sound to me that, the program `copilot` is still running in the background, just detached from the terminal; whereas in reality really the program is stopped.

> If you are deploying Copilot service for the first time in an environment, hitting `Ctrl-C` during deployment will delete the stack.

Reviewer: ```suggestion
If you are deploying Copilot resource for the first time, hitting `Ctrl-C` during deployment will delete the stack.
```

Reviewer 2: piggybacking again 😂  I feel like in this context we can leave the `Copilot` keyword in `Copilot resource` out. All resources deployed by Copilot is Copilot resource.

If you hit 'Ctrl-C' again during a stack deletion, the deletion of CloudFormation stack will continue in the background.

> You can also now use `--detach` flag to skip the progress of Cloudformation Stack Events on your terminal.   

Reviewer 2: I sort of think this way deserve a separate section 💭  It's sort of a separarte feature than `ctrl+c`

## `copilot deploy` enhancements

> ## `detach flag`

Reviewer 2: Similar to the other title, I think we should make this one more descriptive. For example. `Use `--detach` to deploy without waiting`.

> You can now use `--detach` flag for `copilot svc deploy`, `copilot job deploy`, `copilot env deploy` and `copilot deploy` commands to skip the progress of CloudFormation stack events on your terminal.

Reviewer 2: ```suggestion
Typically, after you run any `deploy` commands, Copilot prints the progress to your terminal and waits for the deployment to finish.

Now, if you don't want Copilot to wait, you can use the `--detach` flag. Copilot will trigger the deployment and exit the program, without printing the progress or waiting for the deployment.

The `--detach` flag is available for `copilot svc deploy`, `copilot job deploy`, `copilot env deploy` and `copilot deploy` commands.
```
